src/source/src/table.rs

```diff
@@ -67,75 +69,134 @@ impl TableDmlHandle {
         }
     }
 
-    pub fn stream_reader(&self) -> TableStreamReader {
+    pub fn stream_reader(&self, actor_id: ActorId) -> TableStreamReader {
         let mut core = self.core.write();
-        let (tx, rx) = mpsc::channel(DML_CHUNK_BUFFER_SIZE);
-        core.changes_txs.push(tx);
+        // TODO: use an unbounded channel with permits to limit the buffer size for data chunks.
```

Comment:
Could you please help open an issue for tracking this?

Reply:
Tracking here https://github.com/risingwavelabs/risingwave/issues/10078.